packages/closer/components/custom-pages/CustomPhotoGallery.tsx

```
@@ -0,0 +1,325 @@
+    return rows;
+  };
+
+  const rows = createMasonryRows(images);
+  const baseHeight = 200; // Base height for thumbnails
+
+  return (
+    <section className="max-w-4xl px-4 mx-auto flex flex-col gap-[60px]">
+      <div className="flex flex-col gap-4 text-center">
+        <p
+          className="rich-text"
+          dangerouslySetInnerHTML={{ __html: content?.title }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)